src/shared/GitLab/GitLabConstants.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+
+namespace GitLab
+{
+    public static class GitLabConstants
+    {
+        public const string GitLabBaseUrlHost = "GitLab.com";
+
+        // https://gitlab.com/oauth/applications/207177/edit
+        public const string OAuthClientId = "d8a14250a4d1beacaad67dd6fabaab1e0408b581ca73ae4a76cc7170d3f8afd1";
```

Comment:
Application owned by [me](https://gitlab.com/hickford), replace as you like.

![image](https://user-images.githubusercontent.com/105314/147841415-337be47f-59cf-4865-88b5-df3aeeaf11c3.png)